src/ucp/core/ucp_ep.c

```diff
@@ -1354,16 +1356,22 @@ ucs_status_t ucp_ep_config_init(ucp_worker_h worker, ucp_ep_config_t *config,
              */
             ucs_assert_always(config->tag.rndv.rkey_size <= config->am.max_bcopy);
 
+            rndv_mode     = context->config.ext.rndv_mode;
+            rndv_cap_flag = (((rndv_mode == UCP_RNDV_MODE_GET_ZCOPY) ||
```

Comment:
seems rndv_cap_flag is used only for assertion. i think we can just remove it

Reply:
removed